clang/lib/Format/TokenAnnotator.cpp

```diff
@@ -164,6 +167,12 @@ class AnnotatingParser {
                TT_OverloadedOperatorLParen))) {
         return false;
       }
+      FormatToken *ClosingParen = nullptr;
+      if (Previous.Previous->is(tok::kw_operator) &&
+          isFunctionDeclarationName(Style.isCpp(), *Previous.Previous, Line,
+                                    ClosingParen)) {
```

Comment:
It seems that the following would suffice:
```suggestion
      if (Previous.Previous->is(tok::kw_operator) &&
          CurrentToken->is(tok::l_paren)) {
```